.github/workflows/build.yml

```diff
@@ -134,7 +134,7 @@ jobs:
         # This builds the wire-swift-runtime, verifies that generated protos match what's currently
         # checked into the repo both for the runtime itself and for its tests, and runs the tests.
         run: |
-          ./gradlew :wire-runtime-swift:build --stacktrace --warning-mode all
```

Comment:
@dnkoutso please put them back next time. They help when things go wrong, and I don't think we gain much by taking them out?

Reply:
its extremely hard to scroll through to find a build failure like I did include them. 

Just take a look at a sample here:

<img width="1840" alt="Screenshot 2023-12-20 at 12 39 04 PM" src="https://github.com/square/wire/assets/310370/2034660e-caba-4a2a-8a5f-7ec55c9174cf">

Reply:
`ctrl+f` mate